specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -14011,126 +17307,134 @@
         }
       }
     },
-    "PerfMonResponse": {
-      "description": "Performance monitor API response.",
-      "type": "object",
-      "properties": {
-        "code": {
-          "description": "The response code.",
-          "type": "string"
-        },
-        "message": {
-          "description": "The message.",
-          "type": "string"
-        },
-        "data": {
-          "$ref": "#/definitions/PerfMonSet",
-          "description": "The performance monitor counters."
-        }
-      }
-    },
-    "PerfMonSample": {
-      "description": "Performance monitor sample in a set.",
+    "ProcessModuleInfo": {
+      "description": "Process Module Information.",
       "type": "object",
-      "properties": {
-        "time": {
-          "format": "date-time",
-          "description": "Point in time for which counter was measured.",
-          "type": "string"
-        },
-        "instanceName": {
-          "description": "Name of the server on which the measurement is made.",
-          "type": "string"
-        },
-        "value": {
-          "format": "double",
-          "description": "Value of counter at a certain time.",
-          "type": "number"
-        },
-        "coreCount": {
-          "format": "int32",
-          "description": "Core Count of worker. Not a data member",
-          "type": "integer"
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
         }
-      }
-    },
-    "PerfMonSet": {
-      "description": "Metric information.",
-      "type": "object",
+      ],
       "properties": {
-        "name": {
-          "description": "Unique key name of the counter.",
-          "type": "string"
-        },
-        "startTime": {
-          "format": "date-time",
-          "description": "Start time of the period.",
-          "type": "string"
-        },
-        "endTime": {
-          "format": "date-time",
-          "description": "End time of the period.",
-          "type": "string"
-        },
-        "timeGrain": {
-          "description": "Presented time grain.",
-          "type": "string"
-        },
-        "values": {
-          "description": "Collection of workers that are active during this time.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/PerfMonSample"
-          }
+        "properties": {
+          "description": "ProcessModuleInfo resource specific properties",
+          "properties": {
+            "baseAddress": {
+              "description": "Base address. Used as module identifier in ARM resource URI.",
+              "type": "string"
+            },
+            "fileName": {
```

Comment:
Is your comment about "Base address"?  If so, then I don't know how to clarify it further: It's the module's base address (like what you'd find in windbg, cdb, etc.).